enhancements/network/router-ca-configmap.md

```diff
@@ -0,0 +1,201 @@
+---
+title: router-ca-configmap
+authors:
+  - "@Miciah"
+reviewers:
+  - "@danehans"
+  - "@deads2k"
+  - "@frobware"
+  - "@ironcladlou"
+  - "@knobunc"
+approvers:
+  - "@deads2k"
+  - "@ironcladlou"
+  - "@knobunc"
+creation-date: 2019-11-20
+last-updated: 2019-11-20
+status: implemented
+see-also:
+replaces:
+superseded-by:
+---
+
+# `router-ca` configmap
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+- Do consumers of `router-ca` care about non-default IngressControllers?
+- Should the ingress operator publish `router-ca` if a non-default
+  IngressController has an operator-generated certificate?
+- Should the operator publish `router-ca` even if the default IngressController
+  has a custom default certificate?
+- If so, should the operator publish the default certificate itself to
+  `router-ca`?
+- If the operator publishes custom default certificates to `router-ca`,
```

Comment:
This bullet point seems to summarize the need for a single well-known reference point for components to collect the relevant cert(s) that allow them to communicate with other components on the cluster.

I'm very interested in how these questions are answered.  If for some reason a bundle of all relevant certs should not be published, I'd definitely like to see that the alternative is clearly documented.